PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobLocalRunConfigurationProducer.java

```diff
@@ -91,6 +77,25 @@ protected boolean setupConfigurationFromContext(RemoteDebugRunConfiguration conf
                 .orElse(false);
```

Comment:
I agree that Optional makes our code robust. But I think the style is not very friendly for debugging.

Reply:
You're right, fortunately, IDE can help some Lambda function breakpoints, we can use multi-lines style to meet IDE requirements.